### DISCUSSION TITLE
Disable tunneling and file transfer

### DISCUSSION THREAD

**Discussion by syswipe** [gravitational/teleport#8508] [Q&A] [answered]
Hi guys! I want to implement a bastion host using teleport in my infrastructure. Our security policy requires that bastion host has to allow only ssh connection. Users shouldn't be able to transfer files or make ssh tunnels throw the Bastion. Are there any options in Teleport to satisfy these requirements? Maybe in the Enterprise version? We even ready to buy one :)

**Comment by Valien** [comment, marked as answer]
`SCP` is an integral part of the OpenSSH protocol. There are still many other ways around this to exfiltrate data if a sure really wants to do so. 

With Teleport you have the ability to audit everything that happens. So if a user does SCP then you'll see all the traffic and logs of that interaction.
